florist/app/jobs/page.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+import { ReactElement } from "react/React";
+import useSWR from 'swr';
+import ClientImports, {fetcher, server_address_base, valid_statuses} from '../client_imports';
+
+export default function Page(): ReactElement {
+    const status_components = Object.keys(valid_statuses).map(key => <Status status={key}/>);
+    return (
+        <div className="mx-4">
+            <h1> Job Status </h1>
+            {status_components}
+        </div>
+    );
+}
+
+function Status({ status }) : ReactElement {
+    const endpoint = server_address_base.concat("api/server/job/").concat(status);
+    const {data, error, isLoading} = useSWR(endpoint, fetcher, {refresh_interval: 1000});
+    if (error) return <span> </span>
+    if (isLoading) return <span> </span>
```

Comment:
Make a note to self to add a loading gif as part of your follow up PR.

Reply:
Added to the ticket!